static/app/components/replays/replayContext.tsx

```diff
@@ -354,23 +307,23 @@ export function Provider({
         privateSetCurrentTime(offsetMs);
       }
       if (highlightArgs) {
-        highlight(highlightArgs);
+        addHighlight(highlightArgs);
         setTimeout(() => {
-          clearAllHighlightsCallback();
-          highlight(highlightArgs);
+          clearAllHighlights();
+          addHighlight(highlightArgs);
         });
       }
       didApplyInitialOffset.current = true;
     }
   }, [
-    clearAllHighlightsCallback,
+    clearAllHighlights,
     events,
-    highlight,
+    addHighlight,
     initialTimeOffsetMs,
     privateSetCurrentTime,
   ]);
 
-  useEffect(clearAllHighlightsCallback, [clearAllHighlightsCallback, isPlaying]);
+  useEffect(clearAllHighlights, [clearAllHighlights, isPlaying]);
```

Comment:
Looks like `clearAllHighlights` comes from the hook and is memoized w/ `useCallback` ✅